lemur-build-docker/Dockerfile

```diff
@@ -31,7 +30,7 @@ RUN apt-get update && \
     python -c 'print(" \033[32m BUILDER DONE \033[0m ")'
```

Comment:
I'm getting an error on this line:
```
/bin/sh: 1: python: not found
```

Should it be `python3`?

Reply:
Changed to `python3`, thank you.